lib/licensee/content_helper.rb

```diff
@@ -28,7 +28,7 @@ def length
     # Number of characters that could be added/removed to still be
     # considered a potential match
     def max_delta
-      @max_delta ||= (length * Licensee.inverse_confidence_threshold).to_i
+      @max_delta ||= calculate_max_delta
```

Comment:
Curious why you broke this out into a new method? If rubocop was complaining about line length, you can wrap it in a `begin`/`end` block and put it on its own line.

Reply:
To make it easier to monkeypatch to get the uncached version in a test that varies confidence level (which max_delta needs to respond to). I'll just remove the cache; this is not a heavy calculation.